src/types/qwxdgshell.cpp

```diff
@@ -111,6 +133,9 @@ class QWXdgSurfacePrivate : public QWObjectPrivate
 #endif
         sc.connect(&handle->events.configure, this, &QWXdgSurfacePrivate::on_configure);
         sc.connect(&handle->events.ack_configure, this, &QWXdgSurfacePrivate::on_ack_configure);
+#if WLR_VERSION_MINOR >= 18
+        sc.connect(&handle->surface->events.commit, this, &QWXdgSurfacePrivate::on_commit);
```

Comment:
Why not direct use QWXdgSurface::surface()'s commit?

Reply:
to keep same with other slots. changed to q_func() and QWXdgSurface::commit